### DISCUSSION TITLE
Any way to get automation/envelope data?

### DISCUSSION THREAD

**Discussion by andersstigel** [leolabs/ableton-js#110] [Q&A] [answered]
For clip or track envelopes

E.g. list of parameters with envelopes, envelope timing and paramenter values

**Comment by leolabs** [comment, marked as answer]
I think this is only possible for clips, unfortunately, and Live's API only allows you to fetch one data point in the envelope at a time. For getting the entire envelope, or that of tracks, your best bet would be to parse the project file yourself and get the information that way.
